proto/umee/leverage/v1/query.proto

```diff
@@ -250,3 +256,18 @@ message QueryBadDebtsResponse {
     (gogoproto.nullable) = false
   ];
 }
+
+// QueryMaxWithdraw defines the request structure for the MaxWithdraw gRPC service handler.
+message QueryMaxWithdraw {
+  string address = 1;
```

Comment:
```suggestion
  string supplier = 1;
```

Reply:
This field is currently consistent with the other queries (`QueryAccountSummary`, `QueryAccountBalances`) in naming the field `address`.

It's probably too late to change the others (and they do make sense) so I'd rather keep the queries consistent with each other, even though the transactions use `supplier`

Reply:
In `MsgWithdrawMaximum` you use `supplier`. So why not making it consistent with `MsgWithdrawMaximum`?

Reply:
OK